src/resources/Fields/Fields.ts

```diff
@@ -0,0 +1,44 @@
+import API from '../../APICore';
+import Indexes from '../Indexes/Indexes';
+import Resource from '../Resource';
+import {FieldModel, ListFieldsParams} from './FieldsInterfaces';
+
+export default class Field extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/indexes/fields`;
+
+    create(field: FieldModel) {
+        return this.api.post(Field.baseUrl, field);
+    }
+
+    createFields(fields: FieldModel[]) {
+        return this.api.post(`${Field.baseUrl}/batch/create`, fields);
+    }
+
+    deleteFields(fields: string[]) {
+        return this.api.delete(this.buildPath(`${Field.baseUrl}/batch/delete`, fields));

```

Comment:
I'm not sure the logic I implemented under `buildPath` converts arrays properly :grimacing: Did you try it?

Reply:
😱 good point, I'll review this